### DISCUSSION TITLE
Login to vault using passkeys

### DISCUSSION THREAD

**Discussion by netboy3** [dani-garcia/vaultwarden#4236] [General]
Bitwarden has released Web v2024.1.0 with beta support for using passkeys to encrypt/login to the vault. Is this a client side only feature which we'll gain once web in Vaultwarden is upgraded to 2024.1.0, or does it require server side changes?

**Comment by BlackDex** [comment]
No it isn't. We need to make several changes to the server for this to be supported. But that will probably take a while.

Lets first release a next 1.30.x version before we do that.

**Reply by lukasj98** [reply]
When is it coming approximately?

**Reply by BlackDex** [reply]
No clue. We do not predict or forecast anything in regards to new features.
I would suggest to keep an eye out on PR's created to see if one is having this feature.

But, for that, i think we first need to update the webauthn crate and maybe address some other items in regards to to this.

**Reply by netboy3** [reply]
Adding to that, the actual Bitwarden feature is currently still in beta, only supported on web vault, and requires PRF capable browsers. So even on the client side, this is currently a very limited feature and should probably not be a top priority for Vaultwarden.

**Reply by My1** [reply]
half an eternity later...


seems it has a pretty high adoption, with the primary things not supporting it being safari, ios in general (as browsers are just safari with a skin over there), and Firefox on android specifically.

https://caniuse.com/?search=prf

**Comment by AlexCherrypi** [comment]
Hey, just a friendly reminder.
This feature would really be appreciated :)

**Comment by DocMAX** [comment]
+1

**Comment by pittmanism** [comment]
+1, I just deployed a cloud instance of Bitwarden at work and this feature is amazing. These modern authentication mechanisms have me spoiled!

**Comment by wrohdewald** [comment]
+9999

**Comment by NoahDar** [comment]
Would love to see this get added to Vaultwarden.   I realize it's still beta on Bitwarden but it's a step in the right direction without having to type in long master password!   All of my keys are Yubico Series 5.

**Reply by CodeBradley** [reply]
Same, would love to see this implemented.

**Comment by ToBeHH** [comment]
Link to the corresponding PR: https://github.com/dani-garcia/vaultwarden/pull/5929

**Comment by BobWs** [comment]
Is there any news on login into vault using passkey? Is this still on the roadmap!

**Reply by stefan0xC** [reply]
Nope, it's not on the roadmap because there is no roadmap. But there's already an open PR (that is currently in draft mode) if you check the link the post above yours https://github.com/dani-garcia/vaultwarden/discussions/4236#discussioncomment-13585143

I'm assuming that once #5934 has been reviewed and merged that that PR will be updated as well but until then there probably won't be "any news" regarding this...

**Reply by BobWs** [reply]
> Nope, it's not on the roadmap because there is no roadmap. But there's already an open PR (that is currently in draft mode) if you check the link the post above yours https://github.com/dani-garcia/vaultwarden/discussions/4236#discussioncomment-13585143
> 
> 
> 
> I'm assuming that once #5934 has been reviewed and merged that that PR will be updated as well but until then there probably won't be "any news" regarding this...

Message received loud and clear! Thanks..

**Reply by artichoked1** [reply]
Really hope this gets merged soon!
